inject/src/test/groovy/io/micronaut/context/env/DefaultEnvironmentSpec.groovy

```diff
@@ -81,7 +81,6 @@ class DefaultEnvironmentSpec extends Specification {
         env.getProperty("test.foo", Map.class).get() == [bar: "10", baz: "20"]
     }
 
-    @RestoreSystemProperties
```

Comment:
why are we removing `@RestoreSystemProperties`. These tests are setting system properties.